app/templates/src/app.js

```diff
@@ -1,21 +1,19 @@
 import { DefineMap, route, RoutePushstate } from 'can';
 import 'can-debug#?./is-dev';
 
-route.urlData = new RoutePushstate();
-
 const AppViewModel = DefineMap.extend({
   env: {
-    default: () => ({NODE_ENV:'development'}),
-    serialize: false
-  },
-  message: {
-    default: 'Hello World!',
-    serialize: false
+    default: () => ({NODE_ENV:'development'})
```

Comment:
@matthewp How would this get set when in production? Does steal or autorender do something to set this? Does`env` on the main AppViewModel get automagically mapped to something other than `'development'`?

Reply:
@matthewp Thanks for letting me know that `done-autorender` sets it! -- I'm trying to find the exact line in `done-autorender`... [Is this it](https://github.com/donejs/autorender/blob/21d48c633aac08456e9b278430e361e74bedc365/src/template.txt#L348-L372)? Thanks for your help.

Reply:
In SSR it is here: https://github.com/donejs/autorender/blob/21d48c633aac08456e9b278430e361e74bedc365/src/template.txt#L423-L428

In the client it's not set by anything.